easybuild/easyconfigs/a/ACTC/ACTC-1.1-GCCcore-9.3.0.eb

```
@@ -0,0 +1,34 @@
+easyblock = 'MakeCp'
+
+name = 'ACTC'
+version = '1.1'
+
+homepage = 'https://sourceforge.net/projects/actc'
+description = "ACTC converts independent triangles into triangle strips or fans."
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
```

The `toolchainopts` from `ACTC-1.1-GCCcore-8.3.0.eb` are missing
```suggestion
toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
toolchainopts = {'pic': True}
```